package.json

```diff
@@ -46,7 +46,9 @@
     "@fullcalendar/daygrid": "^6.1.10",
     "@fullcalendar/interaction": "^6.1.10",
     "chart.js": "^3.9.1",
+    "codemirror": "^5.65.16",
```

Comment:
Not sure what this is. I think it'd be better to let whatever dependency relies on this to handle its version instead of declaring it here.